src/native/monodroid/monodroid-tracing.cc

```diff
@@ -22,7 +22,7 @@ void
 MonodroidRuntime::log_traces (JNIEnv *env, TraceKind kind, const char *first_line) noexcept
 {
 	if (!tracing_init_done) {
-		std::lock_guard lock (tracing_init_lock);
```

Comment:
Now I have to wonder why a7b576885624d09c4cc9d65412da2a7558fd3b36 didn't use `xamarin::android::lock_guard` when it was introduced in b90d3ab717.

Reply:
My mistake, I missed it back then.